src/main/java/mate/academy/spring/mapper/impl/response/ShoppingCartResponseMapper.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.spring.mapper.impl.response;
+
+import java.util.stream.Collectors;
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.ShoppingCart;
+import mate.academy.spring.model.Ticket;
+import mate.academy.spring.model.dto.response.ShoppingCartResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ShoppingCartResponseMapper implements
+        DtoResponseMapper<ShoppingCartResponseDto, ShoppingCart> {
+    @Override
+    public ShoppingCartResponseDto toDto(ShoppingCart shoppingCart) {
+        ShoppingCartResponseDto shoppingCartResponseDto = new ShoppingCartResponseDto();
+        shoppingCartResponseDto.setId(shoppingCart.getId());
+        shoppingCartResponseDto.setTicketsId(shoppingCart.getTickets().stream()
+                .map(Ticket::getId)
+                .collect(Collectors.toList()));
+        return shoppingCartResponseDto;
+    }
```

Comment:
forgot about userId

Reply:
I mean thau you added ShoppingCart id, ticketIds, but in shoppingCartResponseDto there is also userId field.

Reply:
I think it's redundant here, because shopping-cart and User have the same Id. In checklist: "When choosing the fields to be returned in your ShoppingCartResponseDto, keep in mind that cartId and userId hold the same value."